### DISCUSSION TITLE
The problem at the start is not enough coins

### DISCUSSION THREAD

**Discussion by Amirjavadzadee** [masterking32/MasterHamsterKombatBot#14] [Q&A] [answered]
Hello.
It's like there is a problem in a part of the code that causes the robot to continue working regardless of the value of ("auto_upgrade_start" , "auto_upgrade_min") variables until the coins run out.

**Comment by masterking32** [comment, marked as answer]
Read this comment:
https://github.com/masterking32/MasterHamsterKombatBot/blob/1244b33af354ba7415593f1c82602acdcd2a030e/main.py#L51
